core/src/main/java/com/microsoft/applicationinsights/internal/config/ConfigurationFileLocator.java

```diff
@@ -147,8 +147,6 @@ private String getConfigurationAbsolutePath(String path) {
         File configFile = new File(path, configurationFileName);
 
         if (configFile.exists()) {
-            InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.INFO, "Found configuration file in: '%s'", path);
```

Comment:
main method, 'getConfigurationFile' prints on exit, the status & the full path of the file if found, which should be enough